### DISCUSSION TITLE
Is this repo tested on PHP 8.4?

### DISCUSSION THREAD

**Discussion by yvermabp** [YOURLS/YOURLS#3837] [Q&A]
We want to add instrumentation to the service, but the pre-requisite for this is PHP 8.0 > version. So we have to upgrade PHP version from 7.2 to 8.0 or 8.4. So Is your services is tested on PHP 8.4 version?
if yes what are the points we would need to take care of?

**Comment by dgw** [comment]
[Not yet](https://github.com/YOURLS/YOURLS/pull/3827).
